evil-commands.el

```diff
@@ -550,7 +550,7 @@ and jump to the corresponding one."
        ((< open close) (goto-char open-pair))
        (t (goto-char close-pair)))))))
 
-(eval-when-compile
+(eval-and-compile
```

Comment:
I don't quite see how this makes a difference to just putting `(require 'flyspell)` there.  If you go for that route, you might as well remove the below `(require 'flyspell)`.

Reply:
This eliminate the ``Warning: the function ‘flyspell-overlay-p’ might not be defined at runtime.`` byte compilation warning message while running ``make``. But yes, you are right, it's better to simply use ``(require 'flyspell)`` at the top level and remove the the below ``(require 'flyspell)``. I will do this.